frontend/tests-regression/README.md

```diff
@@ -0,0 +1,28 @@
+# Regression tests
+
+<p align="center">
+
+  <a href="https://3Drepo.com" target="_blank">
+    <img alt="3D Repo Local-logo" height="70" alt="3D Repo Local Logo" src="https://3drepo.com/wp-content/uploads/2018/06/3D-Repo-Logo-Blue.png"/>
+  </a>
+</p>
+<p align="center">
+    <a href="https://staging.dev.3drepo.io/docs/">Docs</a>
+</p>
+
+The regression tests for 3drepo are meant to be a safeguard against any bugs that might be reappear. Its based on 3drepos tests before pushing out a majore release
```

Comment:
I've noticed a few typos in this file. I'm not sure how important it is or if we care though